remix-gql-tailwind/app/components/UserProfile/UserProfile.view.tsx

```diff
@@ -101,7 +101,7 @@ function UserProfileView({
             <TwitterIcon className={styles.icon} />
             <a
               className={styles.link}
-              href={`https:/twitter.com/${twitterUsername}`}
+              href={`//twitter.com/${twitterUsername}`}
```

Comment:
Twitter has a known protocol and we don't need to let the browser guess it.

```suggestion
              href={`https://twitter.com/${twitterUsername}`}
```